frontend/frontend.js

```diff
@@ -0,0 +1,37 @@
+/*
+This is a promise of extracting the response body as JSON format from the server, 
+and then, calss the populateDescription function  
+*/
+function processFetchedData(response) {
+  const serverResponse = response.json();
+  serverResponse.then(populateDescription);
+}
+
+// This function is to put description and references in html
+function populateDescription(data) {
+  document.getElementById("description").innerText = data.description;
```

Comment:
you should rename this function

```suggestion
function populateFunActivities(data) {
  document.getElementById("description").innerText = data.description;
```

Reply:
I have changed the function name